sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/tests/prerequisites/params.template.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Shouldn't need this file, right?

Reply:
I am using this file in setup.ps1. Basically I read the params.template.json file, and I update it with the proper parameters provided by the user, then output the results in a new file named params.<username>.ps1. 

This logic is in setup.ps1 in the 'Write-Params' function.

Reply:
This params file is then used with the 'az deployment group create' command to create the the resources. I found it easier to provide a params file rather than providing each of these parameters inline.